ci-operator/step-registry/upi/conf/vsphere/ova/upi-conf-vsphere-ova-commands.sh

```diff
@@ -70,8 +71,12 @@ for i in "${!DATACENTERS[@]}"; do
         wait "$!"
     fi
 
-    hw_versions=(15 17 18 19)
+    hw_versions=(15 17 18 19 20)
```

Comment:
I read the logic, seem like it is uploading ova with hw version in order, can you help point me how did we randomly pick up hw version in the testing?